mondrian/src/main/java/mondrian/tui/MockHttpServletRequest.java

```diff
@@ -14,8 +14,8 @@
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.util.*;
-import javax.servlet.*;
-import javax.servlet.http.*;
+import jakarta.servlet.*;
+import jakarta.servlet.http.*;
 
```

Comment:
Import the specific classes instead of *. I know it was already there before but, just to follow the convention

Reply:
I have imported specific packages instead of all packages